index.bs

```diff
@@ -444,9 +459,9 @@ When an {{XRSession}} is shut down the following steps are run:
   1. Remove |session| from the [=list of inline sessions=].
   1. [=Reject=] any outstanding promises returned by |session| with an {{InvalidStateError}}, except for any promises returned by {{XRSession/end()}}.
   1. If no other features of the user agent are actively using them, perform the necessary platform-specific steps to shut down the device's tracking and rendering capabilities. This MUST include:
-    - Releasing [=exclusive access=] to the [=/XR device=].
-    - Deallocating any graphics resources acquired by |session| for presentation to the [=/XR device=].
-    - Putting the [=/XR device=] in a state such that a different source may be able to initiate a session with the same device.
+    - Releasing [=exclusive access=] to the [=XRSession/XR device=] if |session| is an [=immersive session=].
+    - Deallocating any graphics resources acquired by |session| for presentation to the [=XRSession/XR device=].
+    - Putting the [=XRSession/XR device=] in a state such that a different source may be able to initiate a session with the same device.
```

Comment:
I figure it was handwavy enough about "putting it in a state where you can request another session" since that's never not the case for inline sessions, but I made it explicit.